src/main/java/org/broadinstitute/consent/http/models/UserUpdateFields.java

```diff
@@ -1,5 +1,7 @@
 package org.broadinstitute.consent.http.models;
 
+import static com.google.common.base.Objects.equal;
```

Comment:
Even in this case, I wouldn't use a static import in non-test code. Also there is `Objects.equals()` and `Objects.hash()` in `java.util.Objects`. I would prefer using JDK APIs over google ones when possible.